hphp/runtime/ext/mail/config.cmake

```diff
@@ -0,0 +1,6 @@
+HHVM_DEFINE_EXTENSION("mail"
+  SOURCES
+    ext_mail.cpp
+  EXTENSION_LIBRARY
+    ext_mail.php
+)
```

Comment:
I was going to say that the `ext_process` dependency here can be removed after #5682 is merged, but it appears I had eliminated this dependency before I typed up the configs in the first place.